apps/site/src/utils/index.ts

```diff
@@ -4,6 +4,15 @@ import { useAtom } from 'jotai';
 import { useCallback, useEffect, useMemo } from 'react';
 import { addressAtom, userAtom } from '../atoms';
 
+export const config = () => {
+  fcl.config({
+    'accessNode.api': process.env.NEXT_PUBLIC_ACCESS_NODE,
```

Comment:
let's set default values for these so that the environment variables are not required / the app will still work without much configuration.